<?xml version="1.0"?>
<launch>
  <arg name="id" default="bidder3" />
  <include file="$(find talmech)/launch/agent.launch" >
    <arg name="id" value="$(arg id)" />
    <arg name="role" value="bidder" />
    <arg name="params" value="$(find murdoch)/config/$(arg id)_params.yaml" />
  </include>
  <arg name="params" default="$(find murdoch)/config/$(arg id)_executor_params.yaml"/>
  <group ns="$(arg id)">
    <group ns="murdoch">
      <node name="task_executor_node" pkg="murdoch" type="task_executor_node" output="screen" launch-prefix="valgrind">
        <rosparam file="$(arg params)" command="load" />
      </node>
    </group>
  </group>
</launch>
